tests/test_filelock.py

```diff
@@ -0,0 +1,29 @@
+import time
+import os
+
+import pytest
+
+from datasets.utils.filelock import FileLock, Timeout
+
+
+def test_filelock(tmpdir):
+    lock1 = FileLock(tmpdir / "foo.lock")
+    lock2 = FileLock(tmpdir / "foo.lock")
+    timeout = 0.01
```

Comment:
Are you really willing to include a test that increases time latency by 0.01?

Reply:
I think that's ok